templates/htmx-adventure-card.html

```diff
@@ -0,0 +1,44 @@
+<div class="adventure grid bg-white shadow-md rounded-lg transition-shadow hover:shadow-lg outline-blue-400 hover:outline hover:outline-2"
+    data-lang="{{adventure.language or current_language().lang}}"
+    _="on removeMe remove me">
+    <div class="py-4 px-6">
+        {% if adventure["creator"] == user["username"] %}
+        <h2 class="name underline cursor-pointer"
+            _="on click window.open('/for-teachers/customize-adventure/{{adventure.id}}', '_self')">{{adventure.name}}
```

Comment:
Regarding our discussion yesterday I think this is the kind of stuff where Hyperscript shines. Simple, short scripts that don't manipulate state.